kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -2243,11 +2243,25 @@ def _do_fencer_vg(vg, res):
+            lvm_cleanup_timeout = max(cmd.storageCheckerTimeout * 2, 30)
 
-            if lvm.check_vg_status(vg, cmd.storageCheckerTimeout, True)[0] is False:
-                lvm.drop_vg_lock(vg)
-                lvm.remove_device_map_for_vg(vg, keep_device_map=[self.sblk_health_checker.get_record_vm_device_map(vg)])
+            def _lvm_cleanup():
+                try:
+                    lvm.remove_partial_lv_dm(vg)
+                    if lvm.check_vg_status(vg, cmd.storageCheckerTimeout, True)[0] is False:
+                        lvm.drop_vg_lock(vg)
+                        lvm.remove_device_map_for_vg(vg, keep_device_map=[self.sblk_health_checker.get_record_vm_device_map(vg)])
+                except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)